eng/scripts/generate_from_source_pom.py

```diff
@@ -52,6 +52,18 @@
 
 sdk_string = "/sdk/"
 
+# Exclude artifacts list when build from source.These artifacts may be not compatible with current dependencies.
+exclude_artifacts_list = [
+    'azure-spring-cloud-appconfiguration-config',
+    'azure-spring-cloud-appconfiguration-config-web',
+    'azure-spring-cloud-feature-management-web',
+    'azure-spring-cloud-starter-appconfiguration-config',
+    'azure-spring-cloud-test-appconfiguration-config',
```

Comment:
We don't depend on the app configuration libs in Spring Cloud Azure project. So we don't need to exclude them for the spring ci to work correctly?

Reply:
Yes, but it will cause an exception in ci jobs.

Reply:
Which ci?

Reply:
I remember they are configuration ci, core ci